style.css

```diff
@@ -0,0 +1,49 @@
+.block1{
+	border: 7px solid Indigo;
+	width: 400px;
+	margin:0 0 10px 0 
+}
+.block2{
+	border: 7px solid BlueViolet;
```

Comment:
border need to be smaller, need to add paddings for showing background colour and border, now I can see only border without background

the same with `block3` and `block1`